Packs/TwitterIOCHunter-FullDailyFeed/Integrations/TwitterIOCHunterFullDailyFeed/TwitterIOCHunterFullDailyFeed.py

```diff
@@ -1,10 +1,14 @@
 import demistomock as demisto  # noqa: F401
+import requests
 from CommonServerPython import *  # noqa: F401
 
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
 
-def test_module(client):
-    result = client._http_request('GET', '/')
-    if result:
+
+def test_module(url):
+    result = requests.get(url)
```

Comment:
Any reason this was changed to requests and not client._http_request ?

Reply:
Change was to compare status code. But I will change it to use client per your request.